docs/tutorials/acme/dns-validation.rst

```diff
@@ -36,27 +36,16 @@ You can read more about the Issuer resource in the :doc:`Issuer reference docs <
          name: letsencrypt-staging
 
        # ACME DNS-01 provider configurations
-       dns01:
-
-         # Here we define a list of DNS-01 providers that can solve DNS challenges
-         providers:
-
-           - name: prod-dns
-             clouddns:
-               # A secretKeyRef to a google cloud json service account
-               serviceAccountSecretRef:
-                 name: clouddns-service-account
-                 key: service-account.json
-               # The project in which to update the DNS zone
-               project: gcloud-prod-project
-
-           - name: cf-dns
-             cloudflare:
-               email: user@example.com
-               # A secretKeyRef to a cloudflare api key
-               apiKeySecretRef:
-                 name: cloudflare-api-key
-                 key: api-key.txt
+       solvers:
+       - dns01:
+           # Here we define a list of DNS-01 providers that can solve DNS challenges
+           cloudflare:
+           email: <inert your cloudflare host email here>
+           # !! Remember to create a k8s secret before
+           # kubectl create secret generic cloudflare-api-key
+           apiKeySecretRef:
```

Comment:
Looks like the indentation here is incorrect (needs a couple extra spaces for all the cloudflare config!)

Reply:
added